src/components/jacobian_component/jacobian.sh

```diff
@@ -25,11 +25,15 @@ setup_jacobian() {
     # Copy run scripts
     cp ${InversionPath}/src/geoschem_run_scripts/run_jacobian_simulations.sh jacobian_runs/
     sed -i -e "s:{RunName}:${RunName}:g" \
-           -e "s:{InversionPath}:${InversionPath}:g" jacobian_runs/run_jacobian_simulations.sh
+           -e "s:{InversionPath}:${InversionPath}:g" \
+           -e "s:{EndDate}:${EndDate}:g" \
+           -e "s:{ReDoJacobian}:${ReDoJacobian}:g" jacobian_runs/run_jacobian_simulations.sh
+
```

Comment:
Should this just be moved into run_jacobian() rather than having the same sed command in both run_jacobian and setup_jacobian?

Reply:
Yes, fixed